### ISSUE TITLE
no errors are shown in VSCode 1.8.0

### ISSUE THREAD

**Issue by bodyflex** [flowtype/flow-for-vscode#64]
I am not getting any errors in VSCode 1.8.0 with version 276.3.0. running flow in a terminal does output flow errors normally.

here's my `.flowconfig`:
```
[ignore]
./node_modules/config-chain/test/*
./node_modules/npm/node_modules/config-chain/test/*
./node_modules/npm/test/fixtures/config/*

[include]

[libs]
src/types.js

[options]
module.name_mapper='.*\(.css\)' -> 'empty/object'
```

**Comment by qdwang** [comment]
I am not getting any errors either.  My vscode version is 1.8.1

**Comment by orta** [comment]
What are you seeing in the Flow output panel?

e.g.
![screen shot 2016-12-26 at 10 45 08 am](https://cloud.githubusercontent.com/assets/49038/21483920/667c0ff2-cb58-11e6-9f81-91e4292103fd.png)

**Comment by bodyflex** [comment]
I don't think I have that option:
<img width="1242" alt="screen shot 2016-12-27 at 11 35 33" src="https://cloud.githubusercontent.com/assets/7894563/21496644/a4831a36-cc28-11e6-9c90-c1e994b7ace5.png">

**Comment by orta** [comment]
The activation events are any files that are javascript of javascriptreact

```json
    "activationEvents": [
        "onLanguage:javascriptreact",
        "onLanguage:javascript"
    ],
```

So if that's not showing, and you're in one of those files it must be crashing before it gets there. Can you run the command "Developer: Toggle Developer Tools" and check the console to see if there are crashes?

**Comment by orta** [comment]
Also, you need a `.flowconfig` file in your folder structure.

**Comment by agrcrobles** [comment]
+1

**Comment by orta** [comment]
@agrcrobles please don't just +1 

See all the things above I've said above your comment, can you please let us know that you've tried them all and are still not getting anything.

**Comment by agrcrobles** [comment]
@orta I first got:

> [2017-01-07T12:48:01.523Z] [INFO] flow - [unknown]: Flow server in /home/zetta/...
> [2017-01-07T12:48:01.779Z] [INFO] flow - [init]: Flow server in /home/zetta/...
> [2017-01-07T12:48:01.968Z] [INFO] flow - [not running]: Flow server in /home/zetta/...

Then I fixed it by installing flow-bin globally `npm i -g flow-bin` 

Then I had to move flow version of my project in package.json and in at very bottom in .flowconfig because getting the following message:

> The config specifies version 0.36.0 but this is version 0.37.4

Now, I am getting flow warning in vscode!

**Comment by orta** [comment]
Thanks for the writeup!

I think it's propbably worth making a note about the `usePackagedFlow` in the README - which would have fixed that, https://github.com/flowtype/flow-for-vscode/blob/master/package.json#L40

Unlike in the nuclide version this option isn't easily shown to a user.

**Comment by orta** [comment]
This got a README  fix PR - closing

**Comment by tafelito** [comment]
Hi, I know this is an old issue but I'm just experiencing the same error, I think.

I created a new react native project, I installed flow-for-vscode, set `"flow.useNPMPackagedFlow": true` and then added flow-bin as a devDependency (same version as I have in .flowconfig 0.63.0) 

If I run flow in terminal I get an error and I even see it in the Problems tab. But it doesn't show inline in the editor. If I go to the flow output I see this

![image](https://user-images.githubusercontent.com/5973652/36931037-f5cc904c-1e7b-11e8-8911-fae70b84639a.png)

If I actually double click on the error on the Problems tab, it opens the same file twice and then it does show the error inline

No error
![image](https://user-images.githubusercontent.com/5973652/36931042-1ef9bb0c-1e7c-11e8-8f68-539eb5474352.png)

 Problems Tab
![image](https://user-images.githubusercontent.com/5973652/36931053-524e0e2c-1e7c-11e8-8482-e0ac85a5a705.png)

Inline error when double clicking on error from problems tab
![image](https://user-images.githubusercontent.com/5973652/36931074-aee414e2-1e7c-11e8-9246-7463def9c74c.png)

Any ideas how to fix this?

Thanks
